app/weeklyPicks/page.tsx

```diff
@@ -84,22 +81,26 @@ export default function WeeklyPickForm() {
         window.localStorage.setItem('team', data.type);
       }
 
-      const user = await account.get();
-      const response = await getUserWeeklyPick({
-        userId: user.$id,
-        weekNumber: '6622c75658b8df4c4612',
-      });
-
       const findTeamID = NFLTeams.find(
         (ele) => ele.teamName.toLowerCase() === teamSelect,
       );
 
-      response[user.$id] = { team: findTeamID?.$id, correct: true };
+      let appendNewResult;
+      if (userPick?.search(userId!) && userPick?.search(userId!) != -1) {
+        console.error('You already picked a team');
+      } else {
+        const newObj = `"${userId}":{"team":"${findTeamID?.$id}","correct":true}`;
```

Comment:
I don't think you have prettier installed. It should've complained about the spacing in this value.